pulp_rpm/app/viewsets/acs.py

```diff
@@ -49,50 +49,51 @@ class RpmAlternateContentSourceViewSet(AlternateContentSourceViewSet, RolesMixin
                 "action": ["retrieve"],
                 "principal": "authenticated",
                 "effect": "allow",
-                "condition": "has_model_or_obj_perms:rpm.view_rpmalternatecontentsource",
+                "condition": "has_model_or_domain_or_obj_perms:rpm.view_rpmalternatecontentsource",
```

Comment:
So how does this work exactly - if you have "domain perms" do you have access to everything on the domain, or do the object-level restrictions apply on top of the domain ones?  Just wanting to make sure this name captures the function.

Reply:
I would "assume" that RBAC applies "on top of" domains. However - I'm not forging new paths here, just following the trail laid down in pulp_file. @gerrod3 - can you respond here?